### ISSUE TITLE
Impossible to add multiple files in one request

### ISSUE THREAD

**Issue by absence23** [crowdin/crowdin-java-sdk#7]
SDK don't allow to add multiple files and change file path from original local path
```
    if (parameters.getFiles() != null) {
      params.add(CrowdinApiParameters.FILES + OPEN_BRACKET + parameters.getFiles() + CLOSE_BRACKET, parameters.getFiles());
    }
```

**Comment by ihor-popyk** [comment]
Hello @absence23 
File path you can change in parameter `titles()`

**Comment by absence23** [comment]
Yeah, I found it out, but it's a bit tricky and we stil can't add multiple files
Also I found out that it's impossible to export file with path that not exists on device
Mayby I'm mistaken but it's seem so. If I want to export file with path in Crowdin for example /app/translation.xml and I don't have file with the same path on device it's do nothing
